spec/features/project_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'projects' do
+
+  context 'authorized user' do
+
+    scenario 'successfully updates project' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.